doc/examples/bookstore_setup.rb

```diff
@@ -0,0 +1,37 @@
+require 'nokogiri'          # Make nokogiri available.
+include Nokogiri            # Allow omitting leading Nokogiri::.
```

Comment:
I realize you're likely doing this as an editorial decision, to make lines shorter in the examples.

However, I'm not sure it's worth the character savings, because it means that the examples are no long copy-pasteable into ordinary application code. I would honestly prefer not taking shortcuts like this in order to have clearer examples.

In fact, what would you think of just having the bookstore XML be the context, and dropping the `fresh_bookstore_doc` method? Examples would then just start with

```ruby
doc = Nokogiri::XML::Document(BOOKSTORE_XML)
```

instead of

```ruby
doc = fresh_bookstore_doc
```

which isn't much longer but makes the examples more readable because there is less context the reader needs to understand.

Reply:
Fixed.